app/controllers/facility_statements_controller.rb

```diff
@@ -18,16 +18,13 @@ def initialize
 
   # GET /facilities/:facility_id/statements
   def index
-    @statements =
-      current_facility
-      .statements
-      .order(created_at: :desc)
-      .paginate(page: params[:page])
+    statements = current_facility.cross_billing? ? Statement.all : current_facility.statements
```

Comment:
I'm wondering if this is a good opportunity for turning the cross-facility "facility" into a true null object. I think it's something that @drench has been wanting to do.